workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -427,76 +432,58 @@ def __iter__(self):
                     diagnostics.update(substep())
             yield self._state.time, diagnostics
 
+    def monitor(self, name: str, func):
```

Comment:
GitHub had trouble with this diff... I just added a tab to move the `monitor` function inside the `TimeLoop`. I also added units attributes to the `tendency` and `storage` outputs.

Reply:
`monitor` already used the `self` of TimeLoop/MonitoredTimeLoop, so putting the function inside the class just makes this explicit.